examples/environment.yaml

```diff
@@ -5,40 +5,49 @@ channels:
 dependencies:
```

Comment:
I updated the examples environment to more closely match the environments used in CI. @j-wags @mattwthompson At some point it would be good to talk about what we want to do with this environment - its the user-facing examples environment and at some point it stopped being the environment being used in CI, which was news to me today.

Reply:
We only made that switch during this release, weird things were happening around updating an existing environment and which environment the shell had access to later on.

IIUC users aren't expected to install from an environment themselves, since we distribute an `openff-toolkit-examples` package ourselves.